assets/site/frontpage.sass

```diff
@@ -135,68 +143,109 @@ html.front-page
       vertical-align: top
       margin: -1px 2px 0 0
 
+  .intro-text
+    padding: 0 0 2rem
+
+    h2
+      font-size: 2rem
+
+      &:not(:first-child)
+        padding-top: 2rem
+
+    .audience-list
+      // margin: 1rem auto
+      display: grid
+      grid-template-columns: repeat(3, 1fr)
+      grid-gap: 0 2rem
+
+      strong
+        display: block
+
+      @media (max-width: 32rem)
+        grid-template-columns: auto
+
+    .using-cockpit
+      columns: 2
+      column-gap: 1rem
+
   .blurbs
-    margin: 1rem 0 4rem
+    --grid-column: 1 / -1
+    display: grid
+    grid-template-columns: repeat(2, minmax(10rem, 40rem))
+    grid-gap: 3rem
+    background: #eee
+    padding: 6rem 1.5rem 3rem
+    justify-content: center
+    align-items: stretch
+    clip-path: polygon(0 0, 0 100%, 100% 100%, 100% 32px)
+
+    @media (max-width: $frontpage-content-breakpoint)
+      grid-template-columns: auto
 
     .blurb
-      display: grid
-      align-items: flex-start
+      padding: 2rem
+      border-radius: 1rem
+
+      &:nth-child(odd)
+        position: relative
+        top: -1.5rem
 
       @media (min-width: $frontpage-content-breakpoint)
-        grid-template-columns: 1fr 2fr
-        grid-gap: 2rem 1.5rem
+        // grid-template-columns: 1fr 2fr
+        // grid-gap: 2rem 1.5rem
```

Comment:
I'll remove these comments and the comments below and squash rebase at some point.

This branch is still a pretty heavy WIP. :wink: